new_pages/phylogenetic_trees.qmd

```diff
@@ -23,17 +23,17 @@ sample_data <- read.csv(here::here("data","phylo", "sample_data_Shigella_tree.cs
 
 
 ggtree(tree, layout="circular", branch.length='none') %<+% sample_data + # the %<+% is used to add your dataframe with sample data to the tree
```

Comment:
so ggtree and treeio arent on cran and chuck an error @arranhamlet

Reply:
seems they need to be installed with {BiocManager} from bioconductor

Reply:
Yes, I updated the quarto document with this, lines 47-62. Is it still throwing an error?

Reply:
it did for me until i installed separately

Reply:
Ah, it was because they are needed to generate the header image. I've saved this output as a file now rather than re-generating it everytime.